modules/data.remote/R/call_MODIS.R

```diff
@@ -129,8 +204,12 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     output$lat <- round(output$lat, 4)
     output$lon <- round(output$lon, 4)
     
-    fname <- paste(product, "_", band, "_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
-    fname <- paste0(outfolder, "/", fname)
-    write.csv(output, fname)
+    if (!(is.null(outfolder)))
+    {
+      fname <- paste(product, "_", band, "_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
+      fname <- paste0(outfolder, "/", fname)
```

Comment:
Strictly speaking, it's safer to create paths with `file.path`:
```suggestion
      fname <- file.path(outfolder, fname)
```

Reply:
yes, please change. Dont ever use paste0 unless WITHIN the path where you need to past a name together but done ever past in the sep since this various platform to platform, or well windows to unix at least